pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -320,6 +320,10 @@ const (
 // DiscoverySpec contains details of Discovery members
 type DiscoverySpec struct {
 	corev1.ResourceRequirements `json:",inline"`
+
+	// PodSecurityContext of the component
+	// +optional
+	PodSecurityContext *corev1.PodSecurityContext `json:"podSecurityContext,omitempty"`
```

Comment:
In addition to the pod security context for each component, is it better to define a cluster level pod security context so that users only need to configure the cluster level context if they want all the components to share the same context?